.werft/values.dev.yaml

```diff
@@ -25,6 +25,15 @@ tracing:
   samplerType: const
   samplerParam: "1"
 
+# we hit the "max. 110 pods/node" situation pretty often with our current core-dev setup.
+# the proper way to fix those would be to adjust the CIDR for workload NodePool which is a bit more work.
+# as a workaround, we blow up our RAM requests to trigger scaleup earlier.
+# Note: This only works because we tune down our DaemonSet's requests to near-0 (because DS pods don't trigger scalups!)
+resources:
+  default:
+    # as opposed to 200Mi, the default
+    memory: 500Mi
```

Comment:
Why 500Mi? Is there some kind of calculation behind this akin to https://github.com/gitpod-io/gitpod/issues/4700#issuecomment-876314437

Reply:
No calculation as I assumed assignment of static pods to be too dynamic to reason about.

Let's try. To come as close as possible to the theoretical limit of 33 preview envs are goals are:
 - don't consume too much RAM to drive node count up artificially
 - ensure we never reach "max nr of pods" with static deployments

Thus we make them big enough so that say 100 pods fill up the whole node (we ignore the DaemonSets here because they are quite small).
 => 32Gi / 100 ~ 328Mi
 If we choose `350Mi` we're coming close enough it seems.
 
 WDYT? @csweichel

Reply:
Sounds good to me. Let's do this